jupyterlab_comments/README.md

```diff
@@ -0,0 +1,59 @@
+# jupyterlab_comments
+
+Collaborative notebooks in git
```

Comment:
I would recommend using a more specific description here.

Perhaps something like "Notebook comments in git".